eth/tracers/api.go

```diff
@@ -682,86 +767,50 @@ func (api *API) traceInternalsAndAccounts(ctx context.Context, block *types.Bloc
 	if config != nil && config.Reexec != nil {
 		reexec = *config.Reexec
 	}
-	statedb, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, true, false)
+	statedb, release, err := api.backend.StateAtBlock(ctx, parent, reexec, nil, true, false)
 	if err != nil {
 		return nil, err
 	}
+	defer release()
+
+	// Native tracers have low overhead
 	// Execute all the transaction contained within the block concurrently
```

Comment:
This change makes traceTx not run concurrently anymore. Currently, we only use this API with call2 tracer which is a native tracer and as the comment says that native tracers have low overhead. So I am not sure about the performance impact here. Furthermore, if we choose to do this way, we should remove the `Execute all the transaction contained within the block concurrently` comment here.

Reply:
I got your point, it's a mistake, it should be implemented like `traceBlockParallel`